rdmo/management/assets/js/store/configureStore.js

```diff
@@ -1,74 +1,85 @@
-  const currentStoreId = Cookies.get('storeid')
-  const localStoreId = localStorage.getItem('rdmo.storeid')
+  checkStoreId()
 
-  if (isEmpty(localStoreId) || localStoreId !== currentStoreId) {
-    localStorage.clear()
-    localStorage.setItem('rdmo.storeid', currentStoreId)
-  }
+  const rootReducer = combineReducers({
+    config: configReducer,
+    elements: elementsReducer,
+    imports: importsReducer,
+    pending: pendingReducer
+  })
 
-  if (process.env.NODE_ENV === 'development') {
-    const { logger } = require('redux-logger')
-    middlewares.push(logger)
+  const initialState = {
+    config: {
+      prefix: 'rdmo.management.config'
+    }
   }
 
   const store = createStore(
     rootReducer,
-    applyMiddleware(...middlewares)
+    initialState,
+    applyMiddleware(...configureMiddleware())
   )
 
   // load: fetch some of the data which does not change
-  const fetchConfig = () => store.dispatch(configActions.fetchConfig())
-
-  // load: restore the config from the local storage
-  const updateConfigFromLocalStorage = () => {
-    const ls = {...localStorage}
-    Object.entries(ls).forEach(([lsPath, lsValue]) => {
-      if (lsPath.startsWith('rdmo.management.config.')) {
-        const path = lsPath.replace('rdmo.management.config.', '')
-        let value
-        switch(lsValue) {
-          case 'true':
-            value = true
-            break
-          case 'false':
-            value = false
-            break
-          default:
-            value = lsValue
-        }
-        store.dispatch(configActions.updateConfig(path, value))
+  const fetchConfig = () => {
+    return Promise.all([
+      ConditionsApi.fetchRelations(),
+      CoreApi.fetchGroups(),
+      CoreApi.fetchSettings(),
+      CoreApi.fetchSites(),
+      ManagementApi.fetchMeta(),
+      OptionsApi.fetchAdditionalInputs(),
+      OptionsApi.fetchProviders(),
+      QuestionsApi.fetchValueTypes(),
+      QuestionsApi.fetchWidgetTypes()
+    ]).then(([
+      relations, groups, settings, sites, meta,
+      additionalInputs, providers, valueTypes, widgetTypes]) => {
+        store.dispatch(configActions.updateConfig('relations', relations, false))
+        store.dispatch(configActions.updateConfig('groups', groups, false))
+        store.dispatch(configActions.updateConfig('settings', settings, false))
+        store.dispatch(configActions.updateConfig('sites', sites, false))
+        store.dispatch(configActions.updateConfig('meta', meta, false))
+        store.dispatch(configActions.updateConfig('additionalInputs', additionalInputs, false))
+        store.dispatch(configActions.updateConfig('providers', providers, false))
+        store.dispatch(configActions.updateConfig('valueTypes', valueTypes, false))
+        store.dispatch(configActions.updateConfig('widgetTypes', widgetTypes, false))
       }
-    })
+    )
   }
 
-  let currentSiteId
   // load, popstate: fetch elements depending on the location
```

Comment:
I'm getting a warning about `config.currentSite` when browsing directly to `/mangement/catalogs/` or to `/tasks/` or `/views/` , did it go somewhere else?
```js
Uncaught TypeError: can't access property "id", config.currentSite is undefined
```
Browsing to `/management/catalogs/1/` works and browsing from the link in the interview to the management interface also works.